var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -708,6 +710,14 @@ def configure(self, spec, prefix):
     def post_configure_fixes(self):
         spec = self.spec
 
+        # disable the post-build remangling of rpaths using patchelf
```

Comment:
Can you tell me more about this? Is this something that has been reported/fixed upstream? Would love to decide how TF should do things so we don't have to keep this patch around forever. It becomes difficult to maintain and has to be updated across versions. It also makes it difficult to report issues to Google (they refuse to help if it can't be reproduced outside of Spack). Maybe also guard this with which versions need it (only 2.17+?)

Reply:
This is motivated mostly because the python that's executing there isn't aware of the patchelf because the PATH has been cleaned. Let me see if there's an option to "do better". One would think that Bazel would be setting this correctly during the build process...

Reply:
You would be surprised, bazel even strips python3 from its PATH: https://github.com/tensorflow/tensorflow/issues/62497